ci-operator/step-registry/cucushift/installer/rehearse/vsphere/ipi/static-ip/cucushift-installer-rehearse-vsphere-ipi-static-ip-workflow.yaml

```diff
@@ -0,0 +1,11 @@
+workflow:
+  as: cucushift-installer-rehearse-vsphere-ipi-static-ip
+  steps:
+    pre:
+    - chain: cucushift-installer-rehearse-vsphere-ipi-static-ip-provision
+    post:
+    - chain: cucushift-installer-rehearse-vsphere-ipi-deprovision
+    env:
+      FEATURE_SET: TechPreviewNoUpgrade
```

Comment:
Generally we add `TechPreviewNoUpgrade` ENV to ci job config level

Reply:
ack